docs/releases/2.0.txt

```diff
@@ -193,7 +193,8 @@ Tests
 URLs
 ~~~~
 
-* ...
+* The default :ref:`error handlers<topics-error-handling>` are callables
```

Comment:
This is a backwards-incompatible change rather than a "minor feature'. Perhaps you can explain the benefits and give example "handler400, etc." rather than linking to the other section.